test/configs/BUILD

```diff
@@ -16,7 +16,7 @@ licenses(["notice"])  # Apache 2.0
 
 package(default_visibility = ["//visibility:public"])
 
-load("//container/rules:docker_toolchains.bzl", "toolchain_container")
+load("//rules/container:docker_toolchains.bzl", "toolchain_container")
```

Comment:
Add @bazel_toolchains in all these as well?

Reply:
done